src/components/apps/quickLaunch/QuickLaunchListing.js

```diff
@@ -0,0 +1,408 @@
+                        />
+                    )}
+                </Paper>
+                <ActionsPopper
+                    anchorEl={anchorEl}
+                    baseDebugId={baseDebugId}
+                    embedCodeClickHandler={embedCodeClickHandler}
+                    shareClickHandler={shareClickHandler}
+                    onActionPopperClose={() => setAnchorEl(null)}
+                    quickLaunch={selected}
+                    appId={appId}
+                    systemId={systemId}
+                />
+                <Dialog open={embedDialogOpen} maxWidth="sm" fullWidth={true}>
+                    <DialogTitle>
+                        {t("embedLbl")}{" "}
+                        <IconButton
+                            aria-label={t("close")}
+                            onClick={() => setEmbedDialogOpen(false)}
+                            style={{
+                                position: "absolute",
+                                right: theme.spacing(0.5),
+                                top: theme.spacing(0.5),
+                                margin: 0,
+                            }}
+                        >
+                            <CloseIcon />
+                        </IconButton>
+                    </DialogTitle>
+                    <DialogContent>
+                        <CopyTextArea
+                            debugIdPrefix={build(
+                                baseDebugId,
+                                ids.QUICK_LAUNCH.EMBED_QUICK_LAUNCH
+                            )}
+                            text={embedCode}
+                            multiline={true}
+                        />
+                    </DialogContent>
+                </Dialog>
+                <Dialog open={shareDialogOpen} maxWidth="sm" fullWidth={true}>
+                    <DialogTitle>
+                        {t("shareLbl")}
+                        <IconButton
+                            aria-label={t("close")}
+                            onClick={() => setShareDialogOpen(false)}
+                            style={{
+                                position: "absolute",
+                                right: theme.spacing(0.5),
+                                top: theme.spacing(0.5),
+                                margin: 0,
+                            }}
+                        >
+                            <CloseIcon />
+                        </IconButton>
+                    </DialogTitle>
+                    <DialogContent>
+                        <CopyTextArea
+                            debugIdPrefix={build(
+                                baseDebugId,
+                                ids.QUICK_LAUNCH.SHARE_QUICK_LAUNCH
+                            )}
+                            text={qLaunchUrl}
+                            multiline={true}
+                        />
+                    </DialogContent>
+                </Dialog>
```

Comment:
This 2 above dialogs and their buttons could use some IDs for testing

Reply:
I've noticed that the dialogs added in this PR (and other recent PRs) have a close button to the right of the title, but the dialogs I've added recently don't have a close button.

This, along with the need for IDs in common places, makes me wonder if we could have a standard, general purpose dialog in the `utils` namespace 🤔 

(We already have a standard `ConfirmationDialog`.)